File: ProgramacaoFuncionalRevisao/1.py
def calcular_media(lista):
    return sum(lista) / len(lista)

lista = [21, 4, 5, 6, 10, 20, 30]

media = calcular_media(lista)
lista_filtrada = list(filter(lambda x: x < media, lista))

print(media)
print(lista_filtrada)

File: EstudoJS/Funcoes/1.js
function soma(x,y) {
    return x + y;
}
console.log(soma(5,2))

const somaArrow = (x,y) => x+y;
console.log(somaArrow(8,2))

function media(n1, n2, n3) { 
    function calcularMedia(a, b, c) {
        return (a + b + c) / 3;
    }
    return calcularMedia(n1, n2, n3);
}
console.log(media(10, 7, 8)); 